packages/app-project/stores/UI.spec.js

```diff
@@ -1,93 +1,175 @@
       })
       expect(store.mode).to.equal('dark')
-      expect(setCookieSpy).to.not.have.been.called()
+      expect(setModeCookieSpy).to.not.have.been.called()
     })
 
     it('should not update the cookie on instantiation if there is already one', function () {
       document.cookie = 'mode=light; path=/; max-age=31536000'
       store = UI.create()
-      expect(setCookieSpy).to.not.have.been.called()
+      expect(setModeCookieSpy).to.not.have.been.called()
     })
 
     it('should update the cookie if the store mode does not equal the stored cookie mode', function () {
       store.setLightMode()
-      expect(setCookieSpy).to.not.have.been.called()
+      expect(setModeCookieSpy).to.not.have.been.called()
       store.setDarkMode()
-      expect(setCookieSpy).to.have.been.calledOnce()
+      expect(setModeCookieSpy).to.have.been.calledOnce()
+    })
+  })
+
+  describe('dismissedAnnouncementBanner', function () {
+    xit('should contain a dismissedAnnouncementBanner property', function () {
+      expect(store.dismissedAnnouncementBanner).to.be.ok()
+    })
+
+    it('should default to undefined', function () {
+      expect(store.dismissedAnnouncementBanner).to.equal(undefined)
+    })
+
+    it('should have a `dismissAnnouncementBanner` action', function () {
+      const rootStore = initStore(true, { project: PROJECT })
+      const store = rootStore.ui
+      const expectedValue = stringHash(PROJECT.configuration.announcement)
+      expect(store.dismissedAnnouncementBanner).to.equal(undefined)
+      store.dismissAnnouncementBanner()
+      expect(store.dismissedAnnouncementBanner).to.equal(expectedValue)
+    })
+
+    it('should have a showAnnouncement view', function () {
+      const rootStore = initStore(true, { project: PROJECT })
+      const store = rootStore.ui
+      expect(store.showAnnouncement).to.be.true()
+      store.dismissAnnouncementBanner()
+      expect(store.showAnnouncement).to.be.false()
+    })
+  })
+
+  describe('when using the dismissedAnnouncementBanner cookie', function () {
+    let rootStore
+    let store
+    let setAnnouncementBannerCookieSpy
+
+    beforeEach(function () {
+      rootStore = initStore(true, { project: PROJECT })
+      store = rootStore.ui
+      setAnnouncementBannerCookieSpy = sinon.spy(store, 'setAnnouncementBannerCookie')
+      document.cookie = 'dismissedAnnouncementBanner=; max-age=-99999999;'
+    })
+
+    afterEach(function () {
+      setAnnouncementBannerCookieSpy.restore()
+    })
+
+    it('should not set the cookie on instantiation', function () {
+      expect(setAnnouncementBannerCookieSpy).to.not.have.been.called()
+    })
+
+    it('should not update the cookie on instantiation if there is already one', function () {
+      document.cookie = cookie.serialize('dismissedAnnouncementBanner', 1234567890, {
+        path: `/projects/${PROJECT.slug}`
+      })
+      store = UI.create()
+      expect(setAnnouncementBannerCookieSpy).to.not.have.been.called()
```

Comment:
Looking back over this, this isn't a useful expectation. What this should be testing is not that the specific action isn't called, because it won't ever in this test setup, but rather than on patch, it doesn't update the cookie if the existing parsed one is equal to the one in the store. 

It should be noted that the comparable mode test would need to be updated to make sense as well. That can be done separately.